nltk/tokenize/treebank.py

```diff
@@ -58,7 +59,7 @@ class TreebankWordTokenizer(TokenizerI):
         (re.compile(r'([:,])$'), r' \1 '),
         (re.compile(r'\.\.\.'), r' ... '),
         (re.compile(r'[;@#$%&]'), r' \g<0> '),
-        (re.compile(r'([^\.])(\.)([\]\)}>"\']*)\s*$'), r'\1 \2\3 '),
+        (re.compile(r'([^\.])(\.)([\]\)}>"\'»]*)\s*$'), r'\1 \2\3 '),
```

Comment:
This is to tokenize the period.  Instead of u'do.', you should get [..., u'do', u'.', ...].